org.feichtmeier.Musicpod.json

```diff
@@ -0,0 +1,229 @@
+{
+     "app-id": "org.feichtmeier.Musicpod",
+     "runtime": "org.freedesktop.Platform",
+     "runtime-version": "23.08",
+     "sdk": "org.freedesktop.Sdk",
+     "add-extensions": {
+          "org.freedesktop.Platform.ffmpeg-full": {
+               "directory": "lib/ffmpeg",
+               "version": "23.08",
+               "add-ld-path": "."
+          }
+     },
+     "cleanup-commands": [
+          "mkdir -p /app/lib/ffmpeg"
+     ],
+     "command": "musicpod",
+     "finish-args": [
+          "--share=ipc",
+          "--share=network",
+          "--socket=fallback-x11",
+          "--socket=wayland",
+          "--filesystem=home",
+          "--device=dri",
+          "--socket=pulseaudio",
+          "--socket=session-bus",
```

Comment:
```suggestion
```

Like So: @Feichtmeier  could you commit this change? (Or add me as collaborator to the fork used for the PR